### ISSUE TITLE
现在所有系统都有夜间模式，护眼是的但对于看电影和图片这种事情感觉有些碍眼，增加一个打开自动关闭系统夜间模式的功能可好

### ISSUE THREAD

**Issue by iAPPOpen** [iina/iina#2249]
现在所有系统都有夜间模式，护眼是的但对于看电影和图片这种事情感觉有些碍眼，增加一个打开自动关闭系统夜间模式的功能可好

**Comment by alejx** [comment]
I've found [this one](https://github.com/jenghis/nshift) which uses private framework and it works. So it's technically possible. @saagarjha @lhc70000 Do you think it's a good idea to do so?

**Comment by saagarjha** [comment]
Yeah, I'm well aware of CoreBrightness; I already [have a project that links against it](https://github.com/saagarjha/DarkNight) ;)

I'd be happy to write an implementation for this, but I'll defer to you and @lhc70000 whether this is a feature we want to support.
